lib/ZeebeVariableResolver.js

```diff
@@ -1,9 +1,25 @@
 import { getProcessVariables } from '@bpmn-io/extract-process-variables/zeebe';
 import { BaseVariableResolver } from './base/VariableResolver';
+import { parseIoMappings } from './util/feelUtility';
+
+const HIGH_PRIORITY = 2000;
 
 export class ZeebeVariableResolver extends BaseVariableResolver {
   constructor(eventBus, bpmnjs) {
     super(eventBus, bpmnjs);
     this._baseExtractor = getProcessVariables;
+
+    eventBus.on('variableResolver.parseVariables', (e, context) => {
```

Comment:
The listener is registered twice: once here, and once in the base variable resolver. Is this intended?

Reply:
Yes, this is intended. We still want the behavior of the `baseVariableResolver`, but extend it by parsing the IO mappings first. This is why this one one is registered with `HIGH_PRIORITY`